library/runtime.c

```diff
@@ -0,0 +1,156 @@
+/*
+ *  Runtime detection module for Arm64 CPU feature sets.
```

Comment:
`runtime` without detection could be "runtime library", which we call platform support - I'd prefer a different name for the feature detection module

Reply:
How about `cpuid` or `CPU features detection`? they are used in linux kernel also.

Reply:
It has been changed to `cpuid*`